pom.xml

```diff
@@ -12,6 +12,26 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-netty</artifactId>
+      <version>3.0.9.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-jackson2-provider</artifactId>
+      <version>3.0.9.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-client</artifactId>
```

Comment:
As far as I see, we need this dependency only for ParsingResource.java file, right? However, even if we move the file to test/resources Javadoc generation tries to load the file and throws an error, so let's leave the dependency in the pom.xml file. As the ApiFest Doclet is a tool used to generate mappings (docs) files, it is not a constantly running service, it's not a problem. Just change the scope of the dependency to test.